Common/UnitDefinitions/AreaDensity.json

```diff
@@ -56,6 +56,38 @@
           "Abbreviations": [ "mg/m²" ]
         }
       ]
+    },
+    {
+      "SingularName": "PoundPerSquareFoot",
+      "PluralName": "PoundPerSquareFoot",
+      "BaseUnits": {
+        "L": "Foot",
+        "M": "Pound"
+      },
+      "FromUnitToBaseFunc": "{x} * 4.8824",
+      "FromBaseToUnitFunc": "{x} / 4.8824",
```

Comment:
Use exact calculation, not approximations:
> To convert Pounds per Square Foot (psf) to Kilograms per Square Meter (kg/m²), you multiply the psf value by 0.45359237 (for lbs to kg) and divide by 0.092903 (for ft² to m²), which simplifies to multiplying by approximately 4.8824 (or precisely 4.882427758) because 1 psf ≈ 4.8824 kg/m². 

https://share.google/aimode/0Xs1GL9WYEPX2wERx

So 
```
      "FromUnitToBaseFunc": "{x} * 0.45359237 / 0.092903",
      "FromBaseToUnitFunc": "{x} / (0.45359237 / 0.092903)",
```